### PR TITLE
blueprint config updates

### PR DESCRIPTION
**Pull request by yamini**
Modified default.yml for classify and transform, and added new templates:

**Classify**
- default (PII detectors, with nlp on)
- classify-nlp-off (PII detectors, no nlp)
- classify-regex (regex example)
- classify-all-entities  (classify everything)

**Transform**
- default (redact PII, with nlp on)
- transform-nlp-off (redact PII, no nlp)
- transform-regex (regex example)

Updated Readme files as well.

_Validated in Console_
Models • new-configs 2022-01-20 at 5 57 58 PM

### CODE REVIEW

File: config_templates/gretel/transform/default.yml [hunk 1]
```diff
@@ -16,18 +21,11 @@ models:
...
                 - type: redact_with_char
                   attrs:
                     char: X
-label_predictors:
-  namespace: custom
-  regex:
-    user_id:
-      patterns:
-        - score: high
-          regex: "user_[\\d]{5}"
```
Comment (pimlock) [inline config_templates/gretel/transform/default.yml:33]: Thank you for doing that! I just remembered we had this random regex in our default, which didn't make much sense.

File: config_templates/gretel/transform/default.yml [hunk 2]
```diff
@@ -16,18 +21,11 @@ models:
                   - person_name
                   - credit_card_number
                   - phone_number
-                  - us_social_security_number
```
Comment (pimlock) [inline config_templates/gretel/transform/default.yml:19]: I think `us_social_security_number` should stay, right?

File: config_templates/gretel/transform/transform-nlp-off.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                  - credit_card_number
+                  - phone_number
+                  - us_social_security_number
+                  - email_address
+                  - location
+              transforms:
+                - type: fake
+                - type: redact_with_char
+                  attrs:
+                    char: X
```
Comment (pimlock) [inline config_templates/gretel/transform/transform-nlp-off.yml:32]: I think that for the configs that we use as our blueprints, we should change this `redact_with_char` to something else, like e.g. `hash` or `redact_with_label`.

`redact_with_char` can expose a bit too much information, like e.g. if only one person has a name with 2 letters in it, then after doing `redact_with_char` transform, you can still identify that person.

**Review by pimlock — COMMENTED** [review]
This looks great!

File: config_templates/gretel/transform/default.yml (hunk 2 above)
Comment (yamini) [reply config_templates/gretel/transform/default.yml:19]: yeah, i realized that after i deleted it. will add it back

File: config_templates/gretel/transform/transform-nlp-off.yml (hunk 3 above)
Comment (yamini) [reply config_templates/gretel/transform/transform-nlp-off.yml]: done

**Review by pimlock — APPROVED** [review]

**Pull request merged**